Azure/azure-cli
# {SQL} `az sql elastic-pool`: Add tests

**Related command**
- az sql elastic-pool create
- az sql elastic-pool update
- az sql elastic-pool show

**Description**
Records elastic pool tests for VBS enclave functionality

**Testing Guide**
- az sql elastic-pool create
- az sql elastic-pool update
- az sql elastic-pool show

This checklist is used to make sure that common guidelines for a pull request are followed.
